balancer/pickfirst/pickfirstleaf/pickfirstleaf_ext_test.go

```diff
@@ -851,6 +850,245 @@ func (s) TestPickFirstLeaf_EmptyAddressList(t *testing.T) {
 	}
 }
 
+// TestPickFirstLeaf_HappyEyeballs_TFAfterEndOfList verifies that pickfirst
```

Comment:
Optional: These test names are super long and the comment could simply be `// Test verifies blah`. Here and in the other tests in this PR.